site/content/en/development/e2e/e2e-tests-azure.md

```diff
@@ -37,14 +37,16 @@ export AZURE_LOADBALANCER_SKU=<loadbalancer-sku> # the sku of load balancer (opt
 
 - Test it via  ```kubectl version```
 
-## Run Test
+## Run tests
 
-### Have installed ginkgo
-- Run ```ginkgo ./tests/e2e/ ```
+- Run default tests
 
-    For more usage of ginkgo, please follow [ginkgo](https://github.com/onsi/ginkgo/blob/master/README.md)
+  The following command ensures [gingko](https://github.com/onsi/ginkgo) v2 is installed and then runs default tests.
 
-### Without ginkgo
-- Run ```go test ./tests/e2e/ -timeout 0```
+  ``` make test-ccm-e2e ```
+
+- Run specific tests
+
+  ``` go test -v ./tests/e2e/ -timeout 0 -ginkgo.focus <focus-keyword> --ginkgo.skip <skip-keyword> ```
```

Comment:
Have you checked this with ginkgo v2? It looks there is no ginkgo.focus and ginkgo.skip any more.

Reply:
Yes, with TOT code, go test with such option works.